src/pages/workspaces/workspace/Data.js

```diff
@@ -157,12 +163,18 @@ const LocalVariablesContent = ajaxCaller(class LocalVariablesContent extends Com
       ref: this.uploader,
       onDropAccepted: upload
     }, [
-      div({ style: { flex: 'none', display: 'flex', marginBottom: '0.25rem', justifyContent: 'flex-end' } }, [
+      div({ style: { flex: 'none', display: 'flex', alignItems: 'center', marginBottom: '1rem', justifyContent: 'flex-end' } }, [
         linkButton({ onClick: download }, ['Download TSV']),
         !Utils.editWorkspaceError(workspace) && h(Fragment, [
           div({ style: { whiteSpace: 'pre' } }, ['  |  Drag or click to ']),
           linkButton({ onClick: () => this.uploader.current.open() }, ['upload TSV'])
-        ])
+        ]),
+        h(SearchInput, {
+          style: { width: 300, marginLeft: '1rem', borderColor: colors.gray[3] },
```

Comment:
If this is the border color we want on these, perhaps we should roll it into the common component.